filters/other/filter_39_DandelionSprouts_WebPushNotifications/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "filterId": "dandelion_sprouts_web_push_notifications",
+    "id": 39,
+    "name": "Dandelion Sprout's Web Push Notifications",
+    "description": "Aims to remove push notifications on sites. Combines eligible rules from AdGuard Popups Filter's anti-push sections and Fanboy's Notifications Blocking List.",
+    "timeAdded": 1678370714918,
+    "homepage": "https://github.com/DandelionSprout/adfilt",
+    "expires": "4 days",
+    "displayNumber": 4,
+    "environment": "dev",
```

Comment:
It seems that `"environment": "prod"` is right here.

Reply:
No. When add we use `dev` for test purposes, and later change to `prod`.